homeassistant/components/influxdb.py

```diff
@@ -4,22 +4,23 @@
 For more details about this component, please refer to the documentation at
 https://home-assistant.io/components/influxdb/
 """
-from datetime import timedelta
-from functools import partial, wraps
 import logging
 import re
+import queue
```

Comment:
We should use https://docs.python.org/3.6/library/collections.html#collections.deque